doc/develop/microscope.rst

```diff
@@ -43,68 +43,122 @@ The main convention is use the role of each component to indicate the function
+ * light: controls the excitation light of the fluorescence microscope.
+ * laser-mirror: scanner of a confocal microscope
+
+Actuators:
+ * stage: It can have 3 linear axes (x, y, z) and 3 rotational axes (rx, ry, rz). # TODO: stage is where?
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z.
+ * chamber: Manages the pressure and/or sample loading. It must have a "pressure" axis.
+ * pinhole: To change the size of the pinhole in a confocal microscope. It has one axis: d.
+ * sem-stage: the stage of the DELPHI that moves the (whole) sample holder.
+ * align: alignment actuator for the SECOM and DELPHI microscopes.
+   For the SECOM, it must have two axes: "a" and "b".
+   For the DELPHI, it must have two axes: "x" and "y".
+
+Detectors:
+ * se-detector: Secondary electron detector of the SEM (detector).
  * bs-detector: backscattered electron detector of the SEM
  * ebic-detector: EBIC detector of the SEM
- * cl-detector: a cathodoluminescence detector, synchronised with the e-beam
+ * ccd: the main optical pixelated detector (e.g. ccd/cmos/spectral camera).
  * photo-detector(N): a 0D photon detector (eg, PMT or APD). It's currently used
    only in confocal microscopes.
- * spectrometer: A detector to acquire multiple wavelengths information
-   simultaneously. It provides the same interface as a DigitalCamera,
+
+System:
+ * lens: Contains parameters concerning the parabolic mirror and the lens system.
+
+Roles of components found in the SPARC/SPARCv2 system:
+-----------------------------------------------------------------------------------
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample (emitter).
+
+Actuators:
+ * stage: It can have 3 linear axes (x, y, z) and 3 rotational axes (rx, ry, rz). # TODO: stage is where?
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z.
+ * mirror: To engage the parabolic mirror into the beam path.
+   It has two axes in the SPARCv2 system: s (short), l (long).
+ * mirror-xy: To perform the fine adjustments of the position of the parabolic mirror.
+   It has two axes: x and y.
+ * lens-mover: Allows to position lens 1 within the optical path perpendicular to the optical axis.
+   Lens 1 focuses the incoming collimated light. It has an axis: x.
+ * lens-switch: Switches lens 2 between two positions (on: within light path; off: outside of light path).
+   Lens 2 is used to further focus the light coming from lens 1. It has an axis: x or rx.
+ * brightlight: Is used to calibrate the position offset between the two detectors, the grating offset and
+   the focus (mirror) within the spectrograph.
+ * polarization analyzer: It is used to switch the quarter wave plate and the linear polarizer to well
+   specified relative positions to analyze the polarization grade of the emitted light. It has one axis: pol.
+ * quarter wave plate: First polarizer of the polarization analyzer. It has one axis: rz.
```

Comment:
Each role must be one word:
* polarization analyzer: polarization-analyzer
* quarter wave plate: quarter-wave-plate
* linear polarizer: linear-polarizer

BTW, in general we try to keep the names short, to make things fast to type. These names tend to be a little long. I'm fine with them... but if you can think of shorter names, they are welcome ;-)